analyzers/tests/SonarAnalyzer.UnitTest/LiveVariableAnalysis/RoslynLiveVariableAnalysisTest.TryCatchFinally.cs

```diff
@@ -324,19 +324,11 @@ public void Catch_InvalidSyntax_LiveIn()
         {
             /*    Entry 0
              *      |
-             * +----|- TryAndCatchRegion   --------------+
-             * |+---|- TryRegion -+ +-- CatchRegion ----+|
-             * ||  Block 1        | |  Block 3          ||
-             * ||  (empty)        | |  Method(0)        ||
-             * ||   |             | |   |               ||
-             * |+---|-------------+ +---|---------------+|
-             * +----|-------------------|----------------+
-             *      |                  /
-             *      |    /------------+
-             *    Block 3
+             *    Block 1
+             *    Method(0)
```

Comment:
They change the shape of CFG that they generate for this scenario with invalid syntax. 

There's nothing we can do about it. And don't really care.